tests/acceptance/features/apiSpacesShares/copySpaces.feature

```diff
@@ -706,10 +706,12 @@ Feature: copy file
       | permissionsRole | Editor       |
     And user "Alice" has a share "BRIAN-Folder" synced
     When user "Alice" copies file "/textfile1.txt" from space "Personal" to "/BRIAN-Folder" inside space "Shares" using the WebDAV API
-    Then the HTTP status code should be "204"
-    And for user "Alice" the content of the file "/BRIAN-Folder" of the space "Shares" should be "ownCloud test text file 1"
-    And as "Alice" file "/textfile1.txt" should exist
-    And user "Alice" should not have any received shares
+    Then the HTTP status code should be "400"
+    And as "Alice" folder "Shares/BRIAN-Folder/sample-folder" should exist
+    And as "Brian" folder "BRIAN-Folder/sample-folder" should exist
+    But as "Alice" file "Shares/BRIAN-Folder" should not exist
+    And as "Alice" file "Shares/textfile1.txt" should not exist
```

Comment:
Why would this exist if the file is copied inside the BRIAN-Folder in Share drive?

Reply:
didn't get the point.

Reply:
the COPY is not done inside BRIAN-Folder folder but to BRIAN-Folder